### DISCUSSION TITLE
Why isn't version `v3.26.0` on any release platform?

### DISCUSSION THREAD

**Discussion by JustSimplyKyle** [gorilla-devs/ferium#44] [General]
According to the changelog, it seems like it should be released.
https://github.com/theRookieCoder/ferium/blob/main/CHANGELOG.md#v3260

**Comment by theRookieCoder** [comment]
I'm making a couple of changes fixing around 3-5 issues. Then I intend to release `v3.26.0` as a 'QOL update'. You might have noticed I haven't bumped the `Cargo.toml` version yet
